frontend/screens/TestPushScreen.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
```

Comment:
Should delete all notifications-testing-related screens, as we'll be migrating it to a backend server